.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           command: yarn build
       - run:
           name: Run Continuous Integration checks
-          command: yarn lerna run ci-check --scope carbon-components
+          command: yarn lerna run build --scope=carbon-components --include-filtered-dependencies
```

Comment:
Is `ci-check` to `build` change here intentional?

Reply:
Updated!